extensions/iceberg/src/test/java/io.deephaven.iceberg/IcebergToolsTest.java

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg;
+
+import io.deephaven.engine.util.TableTools;
+import io.deephaven.iceberg.util.IcebergCatalog;
+import io.deephaven.iceberg.util.IcebergInstructions;
+import io.deephaven.iceberg.util.IcebergTools;
+import io.deephaven.time.DateTimeUtils;
+import junit.framework.TestCase;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.junit.Test;
+
+import java.util.List;
+
+public class IcebergToolsTest extends TestCase {
+    IcebergInstructions instructions;
+
+    @Override
+    public void setUp() {
+        instructions = IcebergInstructions.builder()
+                .catalogImpl(IcebergInstructions.CATALOG_IMPL.RESTCatalog)
+                .catalogURI("http://rest:8181")
+                .warehouseLocation("s3a://warehouse/wh")
+                .fileIOImpl(IcebergInstructions.FILEIO_IMPL.S3)
+                .s3AccessKeyId("admin")
+                .s3SecretAccessKey("password")
+                .s3EndpointOverride("http://minio:9000")
+                .s3Region("us-east-1")
+                .build();
+    }
+
+    // TODO: discuss how to perform tests since they require a full MiniIO + Iceberg setup
```

Comment:
Just adding a comment so we don't miss removing the TODO before merging.
Also, please look at any unnecessary includes after fixing this.